test/e2e/test-plugins/plugins-api/annotations-sidebar.js

```diff
@@ -118,4 +121,38 @@
 		icon: 'text',
 		render: AnnotationsSidebar
 	} );
+
+	window.annotationsCountingRerenders = 0;
+	const props = {};
+
+	const FORMAT_NAME = 'rerender/counter';
+
+	function countRerender( formats, text ) {
+		if ( text.startsWith( 'RerenderCounter' ) ) {
+			window.annotationsCountingRerenders++;
```

Comment:
It's strange to me that this is being set in a separate file. I have no idea from looking at the other file where it's coming from. Maybe better to more this registration to the test cases?